composer.json

```diff
@@ -438,10 +437,6 @@
   },
   "extra": {
     "patches": {
-      "drupal/backup_migrate": {
```

Comment:
@jbickar FYI

Reply:
Huh. Forgot about that one.

Reply:
![forgot-about-dre](https://user-images.githubusercontent.com/821106/55572766-32a51c80-56bd-11e9-926b-899c53741850.jpg)